### PR TITLE
Update USA_PG_REG9-12.cup

### PR DESCRIPTION
**Pull request by ventus99**
add Texas Soaring and Kansas waypoints and tasks

# The purpose of this change

# The source of the data (for e.g. new frequencies)

### CODE REVIEW

File: data/content/waypoint/region/USA_PG_REG9-12.cup [hunk 1]
```diff
@@ -76,11 +93,116 @@ TP2,TP2,US,2818.917N,09716.367W,14.9m,1,,,,
 TP3,TP3,US,2821.050N,09720.550W,24.1m,1,,,,
 TP4,TP4,US,2818.133N,09722.967W,20.7m,1,,,,
 Triangle Lake,TRIANGLE,US,2934.917N,09556.700W,36.9m,1,,,,Turn Point
+Tsa Gld,TSA GLD,US,3222.954N,09700.763W,203.0m,1,,,,
 Tucker Rd Field,TUCKERRD,US,3220.150N,11120.717W,609.6m,3,,,,Tucker Rd Field
 Wallis,WALLIS,US,2937.883N,09603.917W,40.2m,1,,,,Turn Point
 Waterman,106WATE,US,3221.733N,11127.483W,630.0m,1,,,,Waterman
 White Lake,WhtLake,US,2938.900N,09845.883W,434.3m,2,,,,
 -----Related Tasks-----
+"TexasSA Roundabout CCW","???","Tsa Gld","Sanders Estes Unit","Double A","Ridge Creek","RV Storage","Tsa Gld","???"
```
Comment (lordfolken) [inline data/content/waypoint/region/USA_PG_REG9-12.cup:102]: "???" is a value, Shortnames are parsed. If there is no value intended use ""

**Review by lordfolken — CHANGES_REQUESTED** [review]

File: data/content/waypoint/region/USA_PG_REG9-12.cup [hunk 2]
```diff
@@ -10,58 +11,74 @@ name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
...
+Luscombe,6XS2,US,3220.657N,09711.804W,220.0m,2,7,701.0m,,"Turn Point, #45, Private, 6XS2, LQ8 02/20 2200T PVT, RW width: 90, Fuel: 100LL"
+Marana Regional,056MARA,US,3224.567N,11113.100W,619.0m,5,135,1186.3m,123,Marana Regnl AVQ123.0 AWOS:118.375 RY 03/21:3892x75RY 12/30:6901x99
 Matheson,MATHESON,US,3910.250N,10358.580W,1768.1m,1,,,,"Turn Point, Town"
 Microwave,057MICR,US,3231.167N,11123.883W,733.0m,1,,,,Microwave
 NiwtMt,NIWTMT,US,4003.383N,10533.900W,3440.3m,1,,,,"Turn Point, Niwot MT"
 Orchard,ORCHARD,US,2936.250N,09558.167W,38.1m,1,,,,Turn Point
 Perry Park,PERRYPRK,US,3915.000N,10453.450W,2042.2m,2,160,1737.4m,,"16/34 GATE AT S ENDhwy 83 to Greenland Rd, to N25, to next exit"
 Pescadero,068PESC,US,3211.067N,11123.983W,1079.0m,1,,,,Pescadero
-Pinal Airpark,071PINAL,US,3230.583N,11119.517W,577.0m,5,134,2087.6m,123.050,Pinal Airprk MZJ123.050 AWOS:130.375 RY 12/30:6849x99MGR: SAMUEL SORIA|520-682-4181|(520) 682-4181 X5755
+Pinal Airpark,071PINAL,US,3230.583N,11119.517W,577.0m,5,134,2087.6m,123.05,Pinal Airprk MZJ123.050 AWOS:130.375 RY 12/30:6849x99MGR: SAMUEL SORIA|520-682-4181|(520) 682-4181 X5755
```
Comment (lordfolken) [inline data/content/waypoint/region/USA_PG_REG9-12.cup:76]: I would quote the comment field "Pinal Airprk..." its just safer.

Comment (lordfolken) [reply data/content/waypoint/region/USA_PG_REG9-12.cup:76]: There are several more instances, in general i would quote the comment field.

**Comment by ventus99** [comment]
I have made suggested changes

**Review by lordfolken — APPROVED** [review]

**Pull request merged**